nfstream/flow.py

```diff
@@ -98,6 +100,19 @@ def pythonize_packet(packet, ffi, flow):
                      tunnel_id=packet.tunnel_id)
 
 
+def cdata_dict(cd, ffi):
```

Comment:
* Can you please rewrite it without the use of get_members?
* I guess a dict (with risk as key) is amore suitable struct than a list (Think about converting it to json for storage and also cheking for existence of a specific risk).

Reply:
Yes, indeed! A dict should be more suitable. I will rework the following points

Reply:
Sry, last weeks were a bit troublesome. I will push the fix within this week